Makefile

```diff
@@ -95,6 +95,9 @@ make kubernetes
 # deploy/test all modules in $$CANDIG_MODULES using conda
 make conda
 
+# deploys all modules using Tox
+make tox
```

Comment:
Just asking, are both conda and tox mandatory to build?

Reply:
You can use conda or tox, conda is being left in place for now...